lib/std/fs/test.zig

```diff
@@ -160,6 +160,21 @@ fn testReadLink(dir: Dir, target_path: []const u8, symlink_path: []const u8) !vo
     try testing.expectEqualStrings(target_path, given);
 }
 
+test "relative symlink to parent directory" {
+    var tmp = tmpDir(.{});
+    defer tmp.cleanup();
+
+    var subdir = try tmp.dir.makeOpenPath("subdir", .{});
+    defer subdir.close();
+
+    try subdir.symLink("..\\b.txt", "a.txt", .{});
+
+    var buf: [1000]u8 = undefined;
+    const link_name = try subdir.readLink("a.txt", &buf);
+
+    try testing.expectEqualStrings("..\\b.txt", link_name);
+}
```

Comment:
Is this test meant to be Windows only since it uses `\` as the path separator in the target?

Reply:
You know, at first this was just a straight up mistake due to carelessness, but I thought it was amusing that it actually does pass for POSIX and Windows, but looking at it a third time, a test should really be testing the same thing on all the targets, so I'll go ahead and fix it.